benches/benches/bevy_ecs/world/world_get.rs

```diff
@@ -319,7 +319,7 @@ pub fn query_get_component(criterion: &mut Criterion) {
                 let mut count = 0;
                 for comp in entities
                     .iter()
-                    .flat_map(|&e| query.get_component::<Table>(e))
```

Comment:
This entire benchmark is focused on get_component, so we should probably just remove the entire thing.

Reply:
Fair enough, I'll remove that test!